internal/pkg/platform/gcp/gcp_test.go

```diff
@@ -12,3 +12,73 @@ func TestEmpty(t *testing.T) {
 	// please remove it once any unit-test is added
 	// for this package
 }
+
+// TODO use this mock data for tests
```

Comment:
Committing random curl/JSON output is not something I want to get into the practice of.